web/src/flatpickr.js

```diff
@@ -82,18 +79,12 @@ export function show_flatpickr(element, callback, default_timestamp, options = {
             return true;
         }
 
-        const hotkey = get_keydown_hotkey(e);
-
-        if (!hotkey) {
-            return false;
```

Comment:
(In a jQuery event handler, `return false` is equivalent to the `e.stopPropagation(); e.preventDefault(); return true` we’d reach below.)